Makefile

```diff
@@ -243,6 +243,7 @@ new:
 .PHONY: clean
 clean:
 	@rm -rf $(BUILD_DIR)
+	@rm -rf .venv
```

Comment:
`make clean` is just supposed to clean up the build files, I'm not sure about adding this here